src/features/Articles/Articles.tsx

```diff
@@ -0,0 +1,27 @@
+import { ArticleAPI } from "@/entities/Article";
+import ArticlesStyle from "./Articles.module.scss";
+import Image from "next/image";
+
+export function Articles({ articles }: { articles: ArticleAPI[] }) {
+  return (
+    <div className={ArticlesStyle.articles}>
+      {articles.map((article) => (
+        <div className={ArticlesStyle.article} key={article.title}>
+          {article.urlToImage ? (
```

Comment:
Is there articles without an image inside API?

Reply:
Yes, very often